luigi/parameter.py

```diff
@@ -798,3 +798,34 @@ def parse(self, s):
 
     def serialize(self, x):
         return json.dumps(x, cls=DictParameter.DictParamEncoder)
+
+
+class ListParameter(Parameter):
```

Comment:
Can you add docs? In particular how do you specify elements on CLI/config file. I for one don't know how output of `json.loads([1,2])` looks like.